man/systemd.network.xml

```diff
@@ -3060,6 +3060,15 @@ Token=prefixstable:2002:da8:1::</programlisting></para>
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RetransmitNeighborSolicitationSec=</varname></term>
+
+        <listitem><para>Takes a timespan. Configures the time between retransmitted Neighbor Solicitation messages.
+        Used by address resolution and the Neighbor Unreachability Detection algorithm (see Sections 7.2 and 7.3 of RFC 4861).
+        An unsigned integer ranges 0 to 4294967294. Defaults to 0 seconds.</para>
```

Comment:
This configures what is sent as part of the advertisement, right? So this description should say that: "Configures the minimum time that the client should use between retransmitted Neighbor Solicitation message that is sent in the Router Advertisement message. Takes a timespan with the default unit of seconds. … "

"An integer in the range 0…4294967294."

(The fact that it's unsigned is only meaningful in C. Also, if the value is >= 0, this doesn't really matter.)